src/main/java/algorithms/sorting/radixSort/README.md

```diff
@@ -0,0 +1,45 @@
+# Radix Sort
+
+Radix Sort is a non-comparison based, stable sorting algorithm. 
+We first treat each element as a string with *w* digits (Padding elements that have 
+less than *w* digits).
+
+From the least-significant digit to the most-significant digit, we constantly 
```

Comment:
this sounds very specific. did you base it off somewhere? and depending on implementation, not necessarily 0-9 only.

Perhaps can talk about splitting into segments, and performing counting sort individually on each segment from left to right. Key here is to have the counting sort sub-routine being stable (can explain why its absolutely necessary for it to be stable too!)